extension/src/cli/dvc/reader.ts

```diff
@@ -126,6 +127,10 @@ export class DvcReader extends DvcCli {
       ...flags,
       Flag.JSON
     )
+    if (isEmpty(output)) {
+      return { workspace: { baseline: output as {} } }
```

Comment:
[C] This looks ugly/unnecessary now but makes more sense to do it this way with #2301 coming.